src/components/CallToActionScreen/CallToActionScreen.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <Screen>
+    <Wrapper modal>
+      <Heading class="title">{{ title }}</Heading>
+      <TextParagraph> {{ description }}</TextParagraph>
+    </Wrapper>
+    <Wrapper>
+      <slot/>
+    </Wrapper>
```

Comment:
Adding a ScrollWrapper here:

```suggestion
<ScrollWrapper>
    <Wrapper modal>
      <Heading class="title">{{ title }}</Heading>
      <TextParagraph> {{ description }}</TextParagraph>
    </Wrapper>
    <Wrapper>
      <slot/>
    </Wrapper>
</ScrollWrapper>
```

You can remove the `<div class="button-container">` and let the `<WrappedButton>` to be alone ;)